src/components/CaretSymbol/index.js

```diff
@@ -0,0 +1,10 @@
+import { FaCaretUp, FaCaretDown } from "react-icons/fa";
+
+export const CaretSymbol = (props) => {
+  if (typeof props.value === "boolean") {
+    return props.value ? <FaCaretUp /> : <FaCaretDown />;
+  }
+  if (typeof props.value === "number") {
+    return props.value < 0 ? <FaCaretDown /> : <FaCaretUp />;
+  }
+};
```

Comment:
`export const CaretSymbol = (props) => props.value < 0 ? <FaCaretDown /> : <FaCaretUp />`

then used like this
`<CaretSymbol  value={someValue < 0} />` or `<CaretSymbol  value={boolean} />`

Reply:
niceeeee, didnt know you cound do that

Reply:
eh, i cant get this to work :< i dont understand it very well, maybe you could explain it in the call tonight?